lib/msal-common/src/utils/CryptoUtils.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * @hidden
+ */
+export class CryptoUtils {
+
+    /**
+     * Creates a new random GUID - used to populate state?
+     * @returns string (GUID)
+     */
+    static createNewGuid(): string {
```

Comment:
any plans to update this to UUID version 5?

Reply:
There isn't really a need for us, since version 4 gives us the pseudo-random generator we use UUIDs for. Version 5 will generate this from a hash, meaning it will generate the same UUID given the same input. We don't really need this, so we will stick with version 4.